.github/workflows/ci.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     - name: Install QT
       uses: jurplel/install-qt-action@v4
       with:
-        version: 5.15.2
-        modules: "qtwebengine"
+        version: 6.4.3
+        modules: "qtwebengine qtwebchannel qtpositioning"
```

Comment:
Is the `qtpositioning` really needed?

Reply:
Yes, I'm not sure why, but [the build fails](https://github.com/kiwix/kiwix-build/actions/runs/12473355026/job/34813714594?pr=795#step:11:450) with `Project ERROR: Unknown module(s) in QT: positioning` without it.

Reply:
Pretty sure this is a dependency of qtwebengine... and I suspect there is nothing we can do about that. See https://stackoverflow.com/questions/71087633/getting-the-error-unknown-module-in-qt-positioning-in-qt-creator-when-trying

Reply:
Yes I agree. I didn't think too much about it, but it must be included by default in the other Qt distributions and we are required to include it manually with `jurplel/install-qt-action`.